### PR TITLE
[1LP][WIP]adding some wait-checks for rhos

### PR DESCRIPTION
**Pull request by lkhomenk**
__Fixing__ wait failures on RHOS. when we forgot to wait for provider update or vm provision to complete

### CODE REVIEW

**Comment by lkhomenk** [comment]
failures like "Exception
SSH connection to 10.8.58.167:22 failed, port unavailable" are expected as RHOS faces some network issues with VLANs - FIPs are not accessible at the moment

**Comment by mshriver** [comment]
Is there a resolution for the connection failures? Do you have local test results you can provide? Looks fine otherwise.

**Comment by lkhomenk** [comment]
@mshriver unfortunately  RHOS is still under investigations.

File: cfme/tests/infrastructure/test_instance_analysis.py [hunk 1]
```diff
@@ -208,8 +208,12 @@ def instance(request, local_setup_provider, provider, vm_name, vm_analysis_data)
         image = vm_analysis_data['image']
         vm = VM.factory(vm_name, provider, image)
         request.addfinalizer(vm.delete_from_provider)
-        connect_ip = mgmt_system.get_first_floating_ip()
         provider.refresh_provider_relationships(method='ui')
```
Comment (psav) [inline cfme/tests/infrastructure/test_instance_analysis.py]: A few things here
* We should take the opportunity and refresh via REST I think here
* Secondly, we should perhaps consider adding the assertion of refresh to the method itself

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by lkhomenk** [comment]
RHOS7Ga provider side was fixed. otsuman had some more fixes to this. closing

**Pull request closed without merging**